tests/versioning.go

```diff
@@ -737,7 +1497,7 @@ func (s *VersioningIntegSuite) dispatchActivity(failMode activityFailMode) {
 		if act1state.Add(1) == 1 {
 			switch failMode {
 			case failActivity:
-				return "", errors.New("try again")
+				return "", errors.New("try again") // nolint:goerr113
```

Comment:
cherry-pick/merge #5484 (9da197b0cc) onto the branch and then you don't have to do this